examples/vision/image_captioning.py

```diff
@@ -461,12 +465,16 @@ def test_step(self, batch_data):
                 batch_seq_inp, encoder_out, training=False, mask=mask
             )
 
-            # 5. Update loss and accuracy
-            loss += self.calculate_loss(batch_seq_true, batch_seq_pred, mask)
-            acc += self.calculate_accuracy(batch_seq_true, batch_seq_pred, mask)
+            # 6. Update loss and accuracy
+            caption_loss = self.calculate_loss(batch_seq_true, batch_seq_pred, mask)
+            caption_acc = self.calculate_accuracy(batch_seq_true, batch_seq_pred, mask)
+
+            # 7. Update the batch loss and batch accuracy
+            batch_loss += caption_loss
+            batch_acc += caption_acc
 
-        loss = loss / float(self.num_captions_per_image)
-        acc = acc / float(self.num_captions_per_image)
+        loss = batch_loss
+        acc = batch_acc / float(self.num_captions_per_image)
```

Comment:
In earlier commit, the loss was divided by the number of captions. Since we're interested in accumulative loss, I removed the dividing part. But I'm not so sure which one would be more convenient. Let me know what you think about it.